test/test-component.js

```diff
@@ -256,3 +262,36 @@ describe('Nested Component instance', function() {
     });
   });
 });
+
+
+describe('Rendering exception', function() {
+  var el;
+
+  beforeEach(function(done) {
+    document.body.innerHTML = '';
+    el = document.createElement('breakable-app');
+    el.logError = sinon.spy();
+    document.body.appendChild(el);
+    window.requestAnimationFrame(function() {
+      done();
+    });
+  });
+
+  it('does not prevent component from initializing', function() {
+    expect(el.initialized).to.be.ok;
```

Comment:
ignore me if this is dumb, but the expect docs make it look like you want `to.be.ok()` https://github.com/Automattic/expect.js

Really find that a weird way to describe truthyness (hence having to look it up)... ah well
```
expect(everything).to.be.ok;
```